### ISSUE TITLE
Crashed tests are not showing up as failed

### ISSUE THREAD

**Issue by jacobolofsson** [fredericbonnet/cmake-test-explorer#61]
If a test crashes/hits an assert, the status of the test does not show up as failed, it just changes to "outdated". If the test has previously passed, it just shows as a darker shade of green, which is easy to mistake for passing (IMO).

If you use the native testing UI the crashed test is also not accounted for in the "X/Y tests passed" (example, if you have 100 tests and one crashes, it will say "99/99 test passed (100%)")

I'm using gtest if that makes any difference?

**Comment by fredericbonnet** [comment]
Thanks for reporting! I can't reproduce the test locally (Windows, crashing the test with a segfault with a NULL pointer dereference). Can you tell me more about your setup or share a minimal project? Do you experience the same without gtest?

**Comment by abmantis** [comment]
I have a similar situation, where the test crashes after a something throws a std::runtime_error.

**Comment by rjra100** [comment]
I've reported a similar issue (linked), also on Linux; I think it'll happen with most signal-based failures (segfaults, aborts caused by assertion failures etc). My suspicion is that there may have been a change to the way CTest reports such abnormal exits, so conceivably to repro it requires the "right" version of CTest and an OS that uses signals to implement exits like that. At any rate, I've suggested a fix which worked for me in the report.
